src/Sylius/Bundle/ResourceBundle/Controller/EventDispatcherInterface.php

```diff
@@ -37,6 +37,19 @@ public function dispatch(
     /**
      * @param string $eventName
      * @param RequestConfiguration $requestConfiguration
+     * @param mixed $resources
```

Comment:
Why is it `mixed` instead of `array`?

Reply:
Unfortunatelly it's not always `array` nor `Collection` :/ Take a look at `ResourcesCollectionProviderInterface`

Reply:
Isn't it always 'iterable'?

Reply:
`iterable|ResourceInterface[]`?

Reply:
But `$resources` can be also `ResourceGridView`... I don't think it's iterable :/

Reply:
:-( Indeed.